src/sections/downloads-v3/linux-debian-template.marko

```diff
@@ -25,14 +25,15 @@
     </install-step>
     <install-step num="2" title="Add the Sonarr repository">
         <div>
+            Execute the below three commands in your terminal. <br/>
```

Comment:
I think this reads a little better:

```suggestion
            Execute the three commands below in your terminal. <br/>
```

But also `Execute the commands below in your terminal` would probably be sufficient.

Reply:
I stated three, to be clear that it was three since it spreads over four lines :)